.github/workflows/test_cc.yml

```diff
@@ -51,9 +51,16 @@ jobs:
         TF_INTER_OP_PARALLELISM_THREADS: 1
         LMP_CXX11_ABI_0: 1
         CMAKE_GENERATOR: Ninja
+        CC: clang
+        CXX: clang++
```

Comment:
I guess clang has different symbols from GCC.

```suggestion
```

Reply:
> I guess clang has different symbols from GCC.

Thanks, that works